doc/api/net.markdown

```diff
@@ -359,6 +359,10 @@ The amount of received bytes.
 
 The amount of bytes sent.
 
+### socket.destroyed
+
+A `boolean` value that depicts if the connection is destroyed or not. 
```

Comment:
Look into the source to find out, I think something along the lines of "does not send or receive any more data, cannot be re-established", but I'm no expert on this topic :).